src/operators/rastertypeconversion.md

```diff
@@ -0,0 +1,42 @@
+# RasterTypeConversion
```

Comment:
I guess you also need to add that here: https://github.com/geo-engine/docs/blob/main/src/SUMMARY.md.

Reply:
[f61834f](https://github.com/geo-engine/docs/pull/29/commits/f61834f48103b630a6d859ffcffe669c977bceb0)